geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/util/GeoMesaBatchWriterConfigTest.scala

```diff
@@ -66,4 +66,60 @@ class GeoMesaBatchWriterConfigTest extends Specification {
       ret should equalTo(None)
     }
   }
+
+  "fetchMemoryProperty" should {
```

Comment:
This is an excellent set of tests to show that fetchMemoryProperty is working.  

I'd add another unit test section which demonstrates how to show that a GeoMesaBatchWriterConfig created with GeoMesaBatchWriterConfig.WRITER_MEMORY and the newly supported options works. 

For that you can copy the "respect system properties" whack above, and modify the 'val memoryProp' line and the line where it is checked.  For clarity, I'd remove the other properties.